tools/c7n_org/scripts/orgaccounts.py

```diff
@@ -57,14 +62,17 @@ def main(role, ou, assume, profile, output, regions, active, ignore):
         for k, v in a.get('Tags', {}).items():
             tags.append("{}:{}".format(k, v))
 
+        a['OrgId'] = a['Arn'].split('/')[1]
         if not role.startswith('arn'):
             arn_role = "arn:aws:iam::{}:role/{}".format(a['Id'], role)
         else:
             arn_role = role.format(**a)
         ainfo = {
             'account_id': a['Id'],
             'email': a['Email'],
-            'name': a['Name'],
+            'display_name': a['Name'],
+            'organization_id': a['OrgId'],
+            'name': name.format(**a),
```

Comment:
the only context here is that we should probably render this with the rest of ainfo which is in the config file vs what's in the api response which a user might never see. also we'll want to adjust the jsonschema for c7n-org in cli.py to accept these additional attributes afaicr, accounts are currently set to additionalProperties true but it would good to be explicit about these auto collected attributes.